parse/Parse.cpp

```diff
@@ -29,6 +29,13 @@ namespace std {
 
 namespace parse {
     using namespace boost::xpressive;
+    const sregex CONTENT_TAG_REGEX = bol >> *space >> "#" >> space >> "@content_tag{" >> -*_ >> _n;
```

Comment:
The extra `#` symbol seems unnecessary... why not have the Doxygen in regular comments? That way it can be left in the text during parsing, so that if there is any error that outputs a group of lines surrounding the parse error location, it will include the Doxygen-related text and so be easier to recognize.

Reply:
Ok, I will see if tag_parser.py can be reworked instead.

Reply:
meaning Doxygen requires the first character on a documentation line to be `#` ...? It can being with `// #` ?

Reply:
`content_tag` is an [aliased](https://github.com/freeorion/freeorion/blob/master/doc/Doxyfile.in#L30) special command ([xrefitem](http://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdxrefitem)).
A [docblock](http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html) style that supports special commands is desired.
I chose the triple slash as it is supported by the FOCS parser and does not imply a multi-line style comment as much as `/**  */`.

Doxygen may produce bad links/results if the parsed content does not align favourably with the unmodified source.  Usage of `all_tags.dox` helps alleviate that, but the issue has been temperamental in the past.

Reply:
I didn't understand most of that, but if the result is that you put the Doxygen stuff in a standard-ish comment, it's fine, I think.